LoopKit.xcodeproj/project.pbxproj

```diff
@@ -3599,6 +3695,30 @@
 /* End PBXVariantGroup section */
 
 /* Begin XCBuildConfiguration section */
+		1DEE22FF24A676A300693C32 /* Debug */ = {
+			isa = XCBuildConfiguration;
+			buildSettings = {
+				DEVELOPMENT_TEAM = "";
+				INFOPLIST_FILE = LoopKitHostedTests/LoopKitHostedTests.plist;
+				LD_RUNPATH_SEARCH_PATHS = "$(inherited) @executable_path/Frameworks @loader_path/Frameworks";
+				PRODUCT_BUNDLE_IDENTIFIER = com.loopkit.LoopKitTests;
```

Comment:
[nit] Might want to update the bundle identifier. Probably won't make a difference, but...

Reply:
Same down below.